From 3ac8d44b6a0811a23254e46c23acdc3e8839e18f Mon Sep 17 00:00:00 2001
From: Michael Walle <michael@walle.cc>
Date: Wed, 1 Sep 2021 13:41:37 +0200
Subject: [PATCH 05/32] arm: dts: ls1028a: add an empty /soc

To keep the device tree similar to the linux kernel one, we need to move
all CCSR related devices into the /soc node. To keep the patches easy to
review, we initially add an empty /soc node and populate it piece by
piece.

Signed-off-by: Michael Walle <michael@walle.cc>
---
 arch/arm/dts/fsl-ls1028a-kontron-sl28-u-boot.dtsi | 4 ++++
 arch/arm/dts/fsl-ls1028a.dtsi                     | 7 +++++++
 2 files changed, 11 insertions(+)

diff --git a/arch/arm/dts/fsl-ls1028a-kontron-sl28-u-boot.dtsi b/arch/arm/dts/fsl-ls1028a-kontron-sl28-u-boot.dtsi
index b3861ed98c..fa4c05212a 100644
--- a/arch/arm/dts/fsl-ls1028a-kontron-sl28-u-boot.dtsi
+++ b/arch/arm/dts/fsl-ls1028a-kontron-sl28-u-boot.dtsi
@@ -266,6 +266,10 @@
 	u-boot,dm-pre-reloc;
 };
 
+&soc {
+	u-boot,dm-pre-reloc;
+};
+
 &sysclk {
 	u-boot,dm-pre-reloc;
 };
diff --git a/arch/arm/dts/fsl-ls1028a.dtsi b/arch/arm/dts/fsl-ls1028a.dtsi
index 48f70c10d9..1f562cfdad 100644
--- a/arch/arm/dts/fsl-ls1028a.dtsi
+++ b/arch/arm/dts/fsl-ls1028a.dtsi
@@ -478,4 +478,11 @@
 		compatible = "arm,sp805-wdt";
 		reg = <0x0 0xc000000 0x0 0x1000>;
 	};
+
+	soc: soc {
+		compatible = "simple-bus";
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+	};
 };
-- 
2.30.2

